### PR TITLE
tests: speed-up data generation

### PR DESCRIPTION
**Pull request by jd**
Calling 28800 times `date' is pretty slow. Let's generate that data in a snap
using Python.

### CODE REVIEW

**Comment by jd** [comment]
Pull-request updated, HEAD is now f7511912ab8b41fc76ecb8868aa32eaba06433fc

**Comment by jd** [comment]
Pull-request updated, HEAD is now d84069e0166299e5db0f1c16cd2a584e72dacd43

File: tox.ini [hunk 1]
```diff
@@ -48,7 +48,7 @@ commands =
     {toxinidir}/run-tests.sh {posargs}
     {toxinidir}/run-func-tests.sh {posargs}
 
-[testenv:py35-postgresql-file-upgrade-from-3.1]
+[testenv:py36-postgresql-file-upgrade-from-3.1]
```
Comment (chungg) [inline tox.ini]: i should ask why you change to 3.6 right? i'll ask why... why you change to 3.6 if everything else is 3.5? :)

**Comment by jd** [comment]
Pull-request updated, HEAD is now cc1a6a1db7dbf455898692cc779c9cdc7524df6d

**Comment by jd** [comment]
Pull-request updated, HEAD is now 64e9f56c0a62d6e327e19de5203ffeaa2792ccf9

File: tox.ini (hunk 1 above)
Comment (jd) [inline tox.ini]: Good catch. I needed to change it to test in my env as I don't have Python 3.5 anymore. But I did not check while committing!

Thanks @chungg !

**Pull request merged**